lib/bots/versionbot.ts

```diff
@@ -767,12 +821,19 @@ export class VersionBot extends ProcBot {
 		const name = head.repo.name;
 		const author = prEvent.sender.login;
 		const prAction = event.cookedEvent.data.action;
+		const prLabel: GithubApiTypes.Label = event.cookedEvent.data.label;
 		let committer = author;
 		let lastCommit: GithubApiTypes.Commit;
 		let botConfig: VersionBotConfiguration;
 
-		// Only for opened or synced actions.
-		if (!_.find(['opened', 'synchronize', 'labeled'], (action) => action === prAction)) {
+		// Only for opened, synced or labeling actions.
+		if (!_.find(['opened', 'synchronize', 'labeled', 'unlabeled'], (action) => action === prAction)) {
```

Comment:
I don't have much lodash knowledge, but couldn't `_.includes` (or `_.some`) be used instead of `_.find` ?